File: src/main/java/RobotSpaceExplorator.java
import coreClasses.Controller;
import utils.Utils;

public class RobotSpaceExplorator {
    private static void createConfigFile() {
        Utils.deleteFileIfExists(Utils.defaultLogFileName);
        Utils.createNewFileIfNotExists(Utils.defaultLogFileName);
    }

    public static void main(String[] args) {
        createConfigFile();
        Utils.log("test");

        var controller = new Controller();
        controller.scheduleMissions(Controller.defaultNumberOfSimultaneousMissions);
    }
}

File: src/main/kotlin/RobotSpaceExplorator.kt
import coreClasses.controller.Controller
import utils.Utils

object RobotSpaceExplorator {
    fun createConfigFile() {
        Utils.deleteFileIfExists(Utils.defaultLogFileName)
        Utils.createNewFileIfNotExists(Utils.defaultLogFileName)
    }
}

fun main(args: Array<String>) {
    RobotSpaceExplorator.createConfigFile()
    val controller = Controller()
}
